pkg/bookmarks/bookmarks.go

```diff
@@ -72,3 +89,22 @@ func ReadAll(path string) (map[string]Bookmark, error) {
 
 	return results, nil
 }
+
+type ErrNonExistingBookmark string
```

Comment:
https://github.com/sosedoff/pgweb/pull/201/commits/68db9345071bd806ae7ea0eaaa08019048febbfb gets rid of this err pattern